internal/pkg/service/buffer/storage/local/writer/disksync/disksync.go

```diff
@@ -117,168 +117,141 @@ func NewSyncer(logger log.Logger, clock clock.Clock, config Config, chain chain)
 	return w
 }
 
-// Write writes to the underlying writer and trigger synchronization when the configured Config.BytesTrigger volume is exceeded.
-func (s *Syncer) Write(p []byte) (n int, err error) {
-	if err := s.ctx.Err(); err != nil {
-		return 0, errors.Errorf(`syncer is closed: %w`, err)
+// AddWriteOp increments number of high-level writer operations,
+// for example writing one row of the table is one high-level write operation.
+func (s *Syncer) AddWriteOp(n uint) {
+	s.writeOpsCount.Add(uint64(n))
+}
```

Comment:
Jedina vec, ktoru nevieme merat priamo je pocet high-level write operacii.

Priklad, `CSV Writer (WriteRow) -> Syncer (Write) -> Chain (Write)`:
- CSV Writer zavolame s `WriteRow([]string{....})`.
- A on moze zavolat `Write([]byte)` na dalsej urovni lubovolny pocet krat.
  - K tomu ma este interný buffer, takze pri prvom riadku nezavolá nič,
    a keď sa nazbierajú aspoň `4KB`, tak to pošle ďalej.
  - Je to efektívne čo sa týka rýchlosti, počtu volaní funkcie, atď ...
  - Ale nám to komplikuje počítanie toho, koľko `WriteRow` operácii vlastne čaká.
  - Preto musíme zavolať manuálne na úrovni `CSV Writer`, ktorý posledný vie niečo o nejakom `row ` - metódu `AddWriteOp(1)`, dalej sú to už iba `bytes`.
  
  -----
  
  Potrebujeme to pre `CountTrigger`.